app/assets/stylesheets/components.scss

```diff
@@ -0,0 +1,16 @@
+// Place all the styles related to the components controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.app-certificate-tag-expiring .govuk-tag {
+  background-color: #b10e1e;
```

Comment:
```suggestion
  background-color: $govuk-error-colour;
```

Reply:
Something for Chris T, whether red (interpreted as error) is suitable for expiring, whether it shouldn't be yellow as a warning?